docs/source/contributing_guide.md

```diff
@@ -44,6 +44,10 @@ Install dependencies:
 pip install -r requirements-dev.txt
 ```
 
+```
+pip install -r requirements-docs.txt
+```
+
```

Comment:
you can replace original code snippet above with this one, since `requirements-docs.txt` also includes `requirements-dev.txt`